index.html

```diff
@@ -0,0 +1,108 @@
+<!doctype html>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+
+<body class="body">
+    <div >
+        <div class="top">
+            <h1 class="ROSE"><img width="235px" height="30px" src="rose.png" alt=""></h1>
+            <div class="wholeBurger">
+                <div class="topPartOfTheBurger"></div>
+                <div class="middlePartOfTheBurger"></div>
+                <div class="bottomPartOfTheBurger"></div>
+            </div>
+        </div>
+        <div>
+            <img class="topPhoto" width="375px" height="235px"
+                src="imgonline-com-ua-Resize-urNhT9kARoZ94SZ.jpg">
+        </div>
+        <div class="textAfterTop">
+            <div class="aboutBlog">A few words about this blog platform, Ghost, and how this site was made</div>
+            <div class="smallTextAboutBlog">Why Ghost (& Figma) instead of Medium, WordPress or other options?</div>
+        </div>
+        <div class="lineOfFirstPage"></div>
```

Comment:
this must be implemented using `border` css property. ideally you shouldn't have empty html tags on the page